### PR TITLE
Fix: [pumpio] Wrong variable name in "if" / Some reworked parts

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: pumpio/pumpio.php [hunk 1]
```diff
@@ -188,10 +188,10 @@ function pumpio_connect(&$a) {
 		}
 		$success = $client->Finalize($success);
 	}
-	if($client->exit)
+	if ($client->exit)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 2]
```diff
@@ -204,11 +204,11 @@ function pumpio_connect(&$a) {
 }
 
 function pumpio_jot_nets(&$a,&$b) {
-	if(! local_user())
+	if (!local_user())
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 3]
```diff
@@ -219,7 +219,7 @@ function pumpio_jot_nets(&$a,&$b) {
 
 function pumpio_settings(&$a,&$s) {
 
-	if(! local_user())
+	if (!local_user())
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 4]
```diff
@@ -396,18 +396,18 @@ function pumpio_post_local(&$a, &$b) {
 function pumpio_send(&$a,&$b) {
 
 	if (!PConfig::get($b["uid"],'pumpio','import')) {
-		if($b['deleted'] || $b['private'] || ($b['created'] !== $b['edited']))
+		if ($b['deleted'] || $b['private'] || ($b['created'] !== $b['edited']))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: This condition can be rewritten to 
```php
if (!PConfig::get($b["uid"], 'pumpio', 'import') && ($b['deleted'] || $b['private'] || $b['created'] !== $b['edited'])) {
```

File: pumpio/pumpio.php [hunk 5]
```diff
@@ -449,11 +449,11 @@ function pumpio_send(&$a,&$b) {
 	if (($b['verb'] == ACTIVITY_POST) && $b['deleted'])
 			pumpio_action($a, $b["uid"], $b["uri"], "delete");
 
-	if($b['deleted'] || ($b['created'] !== $b['edited']))
+	if ($b['deleted'] || ($b['created'] !== $b['edited']))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 6]
```diff
@@ -449,11 +449,11 @@ function pumpio_send(&$a,&$b) {
...
 		return;
 
 	// if post comes from pump.io don't send it back
-	if($b['app'] == "pump.io")
+	if ($b['app'] == "pump.io")
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 7]
```diff
@@ -531,22 +531,22 @@ function pumpio_send(&$a,&$b) {
...
+			if ($post_id && $iscomment) {
 				logger('pumpio_send '.$username.': Update extid '.$post_id." for post id ".$b['id']);
 				Item::update(['extid' => $post_id], ['id' => $b['id']]);
 			}
 		} else {
 			logger('pumpio_send '.$username.': '.$url.' general error: ' . print_r($user,true));
 
 			$r = q("SELECT `id` FROM `contact` WHERE `uid` = %d AND `self`", $b['uid']);
-			if (count($r))
+			if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 8]
```diff
@@ -612,13 +612,13 @@ function pumpio_action(&$a, $uid, $uri, $action, $content = "") {
 	else
 		$success = false;
 
-	if($success)
+	if ($success)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 9]
```diff
@@ -612,13 +612,13 @@ function pumpio_action(&$a, $uid, $uri, $action, $content = "") {
...
 		logger('pumpio_action '.$username.' '.$action.': success '.$uri);
 	else {
 		logger('pumpio_action '.$username.' '.$action.': general error: '.$uri.' '.print_r($user,true));
 
 		$r = q("SELECT `id` FROM `contact` WHERE `uid` = %d AND `self`", $b['uid']);
-		if (count($r))
+		if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 10]
```diff
@@ -632,26 +632,26 @@ function pumpio_sync(&$a) {
 	$r = q("SELECT * FROM `addon` WHERE `installed` = 1 AND `name` = 'pumpio'",
 		$plugin);
 
-	if (!count($r))
+	if (!DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 11]
```diff
@@ -632,26 +632,26 @@ function pumpio_sync(&$a) {
...
 		return;
 
 	$last = Config::get('pumpio','last_poll');
 
 	$poll_interval = intval(Config::get('pumpio','poll_interval'));
-	if(! $poll_interval)
+	if (!$poll_interval)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 12]
```diff
@@ -680,12 +680,12 @@ function pumpio_sync(&$a) {
 
 			// check for new contacts once a day
 			$last_contact_check = PConfig::get($rr['uid'],'pumpio','contact_check');
-			if($last_contact_check)
+			if ($last_contact_check)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 13]
```diff
@@ -823,33 +823,33 @@ function pumpio_dounlike(&$a, $uid, $self, $post, $own_id) {
...
+	if (link_compare($post->actor->url, $own_id)) {
 		$contactid = $self[0]['id'];
 	} else {
 		$r = q("SELECT * FROM `contact` WHERE `nurl` = '%s' AND `uid` = %d AND `blocked` = 0 AND `readonly` = 0 LIMIT 1",
 			dbesc(normalise_link($post->actor->url)),
 			intval($uid)
 		);
 
-		if(count($r))
+		if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 14]
```diff
@@ -823,33 +823,33 @@ function pumpio_dounlike(&$a, $uid, $self, $post, $own_id) {
...
 			$contactid = $r[0]['id'];
 
-		if($contactid == 0)
+		if ($contactid == 0)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 15]
```diff
@@ -823,33 +823,33 @@ function pumpio_dounlike(&$a, $uid, $self, $post, $own_id) {
...
 			$contactid = $orig_post['contact-id'];
 	}
 
 	Item::delete(['verb' => ACTIVITY_LIKE, 'uid' => $uid, 'contact-id' => $contactid, 'thr-parent' => $orig_post['uri']]);
 
-	if(count($r))
+	if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 16]
```diff
@@ -890,10 +890,10 @@ function pumpio_dolike(&$a, $uid, $self, $post, $own_id, $threadcompletion = tru
 			intval($uid)
 		);
 
-		if(count($r))
+		if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 17]
```diff
@@ -890,10 +890,10 @@ function pumpio_dolike(&$a, $uid, $self, $post, $own_id, $threadcompletion = tru
...
 			$contactid = $r[0]['id'];
 
-		if($contactid == 0)
+		if ($contactid == 0)
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 18]
```diff
@@ -1093,15 +1093,15 @@ function pumpio_dopost(&$a, $client, $uid, $self, $post, $own_id, $threadcomplet
 				intval($uid)
 			);
 
-			if(count($r))
+			if (DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 19]
```diff
@@ -1308,11 +1308,11 @@ function pumpio_queue_hook(&$a,&$b) {
 	$qi = q("SELECT * FROM `queue` WHERE `network` = '%s'",
 		dbesc(NETWORK_PUMPIO)
 	);
-	if(! count($qi))
+	if (!DBM::is_result($qi))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 20]
```diff
@@ -1321,7 +1321,7 @@ function pumpio_queue_hook(&$a,&$b) {
 			WHERE `contact`.`self` = 1 AND `contact`.`id` = %d LIMIT 1",
 			intval($x['cid'])
 		);
-		if(! count($r))
+		if (!DBM::is_result($r))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Please wrap conditions with braces.

File: pumpio/pumpio.php [hunk 21]
```diff
@@ -1386,7 +1386,7 @@ function pumpio_getreceiver(&$a, $b) {
 
 	if (!$b["private"]) {
 
-		if(! strstr($b['postopts'],'pumpio'))
+		if (!strstr($b['postopts'],'pumpio'))
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards:
- Please wrap conditions with braces.
- Please add a space after commas.

File: pumpio/pumpio.php [hunk 22]
```diff
@@ -433,28 +438,34 @@ function pumpio_send(&$a,&$b) {
...
 	}
 
-	if ($b['verb'] == ACTIVITY_DISLIKE)
+	if ($b['verb'] == ACTIVITY_DISLIKE) {
 		return;
+	}
 
-	if (($b['verb'] == ACTIVITY_POST) && ($b['created'] !== $b['edited']) && !$b['deleted'])
+	if (($b['verb'] == ACTIVITY_POST) && ($b['created'] !== $b['edited']) && !$b['deleted']) {
 			pumpio_action($a, $b["uid"], $b["uri"], "update", $b["body"]);
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Wrong indentation level.

File: pumpio/pumpio.php [hunk 23]
```diff
@@ -433,28 +438,34 @@ function pumpio_send(&$a,&$b) {
...
+	}
 
-	if (($b['verb'] == ACTIVITY_POST) && $b['deleted'])
+	if (($b['verb'] == ACTIVITY_POST) && $b['deleted']) {
 			pumpio_action($a, $b["uid"], $b["uri"], "delete");
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Wrong indentation level.

File: pumpio/pumpio.php [hunk 24]
```diff
@@ -483,35 +494,41 @@ function pumpio_send(&$a,&$b) {
 						'objectType' => "note",
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Wrong indentation level.

File: pumpio/pumpio.php [hunk 25]
```diff
@@ -483,35 +494,41 @@ function pumpio_send(&$a,&$b) {
...
 			$inReplyTo = ["id" => $orig_post["uri"],
 					"objectType" => "note"];
 
-			if (($orig_post["object-type"] != "") && (strstr($orig_post["object-type"], NAMESPACE_ACTIVITY_SCHEMA)))
+			if (($orig_post["object-type"] != "") && (strstr($orig_post["object-type"], NAMESPACE_ACTIVITY_SCHEMA))) {
 				$inReplyTo["objectType"] = str_replace(NAMESPACE_ACTIVITY_SCHEMA, '', $orig_post["object-type"]);
+			}
 
 			$params["object"] = [
 						'objectType' => "comment",
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Wrong indentation level.

File: pumpio/pumpio.php [hunk 26]
```diff
@@ -483,35 +494,41 @@ function pumpio_send(&$a,&$b) {
...
+			}
 
-			if (count($receiver["bcc"]))
+			if (count($receiver["bcc"])) {
 				$params["bcc"] = $receiver["bcc"];
-
+			}
 		 } else {
 			$inReplyTo = ["id" => $orig_post["uri"],
 					"objectType" => "note"];
```
Comment (MrPetovan) [inline pumpio/pumpio.php]: Code standards: Wrong indentation level.

**Comment by annando** [comment]
@MrPetovan Is there something that you want me to do with this PR?

**Comment by MrPetovan** [comment]
I  want you to dance with it fo a while, but I'm not sure that's what you asked for.

**Comment by annando** [comment]
I'm mostly working with my code for some time, mostly :-)

**Pull request merged**